src/_guides/language/language-tour.md

```diff
@@ -186,31 +186,31 @@ This content has moved to [Operators](/language/operators#other-operators).
 
 ## Control flow statements
 
-This content has moved to [Control flow](/language/control-flow).
+This content has moved to [Control flow](/language/loops).
```

Comment:
Most people reading this document aren't going to have read it before, so they don't care that the content was here before and moved.

I would phrase this as something like, "For information on control flow statements, see ..."

Reply:
We [took the language tour down](https://github.com/dart-lang/site-www/pull/4612) and made it unsearchable, so the only people that will end up on that page actually are people who already have the link saved